api/v1/endpoints/queries.sql

```diff
@@ -100,12 +104,11 @@ WITH object_data AS (
     WHERE
         ci.id = $course_instance_id
         AND a.id = $assessment_id
-        AND aar.assessment_id = $assessment_id
 )
 SELECT
     coalesce(jsonb_agg(
         to_jsonb(object_data)
-        ORDER BY assessment_id, id
+        ORDER BY assessment_id, assessment_access_rule_id
```

Comment:
```suggestion
        ORDER BY assessment_access_rule_number, assessment_access_rule_id
```

There is only one `assessment_id`, so sorting by it won't do anything. The `number` column is the best thing to sort by.